src/components/card.js

```diff
@@ -0,0 +1,67 @@
+import React, { Component } from 'react';
+import GithubApiInterface from './../githubApiInterface';
+
+export default class Card extends Component {
+    constructor(props){
+        super(props);
+        this.data = this.props.data;
+        this.content = "";
+        this.link = "";
+        this.userlink = "";
+        this.imageLink = "";
+        this.username = "";
+        this.repoName = "";
+        this.numForks = "";
+        this.numStars = "";
+        this.repoLink = "";
+    }
+
+    componentWillMount(){
```

Comment:
should we call componentDidMount instead? As per https://reactjs.org/docs/react-component.html, componentWillMount should be obsolete

Reply:
This originally wouldn't work as componentDidMount, but I'm glad you pointed it out here, I have changed it to didmount and refactored the card to be stateful, so now it works with better design